### DISCUSSION TITLE
Tauri's onDragDropEvent gives strange y coordinate?

### DISCUSSION THREAD

**Discussion by shi-yan** [tauri-apps/tauri#11141] [Q&A] [answered]
Hello, my app has file drag and drop features. I need to get the location where a file is dropped.

the file drop event has a position, but its coordinates are strange. the y value is often negative, and doesn't make sense to me.

<img width="912" alt="Screenshot 2024-09-25 at 9 44 34 PM" src="https://github.com/user-attachments/assets/74f463ff-279e-4066-8ab2-6aaafc058787">

The document doesn't say anything about its coordinates https://beta.tauri.app/reference/javascript/api/namespacewebview/#dragdropevent ?

**Comment by FabianLars** [comment, marked as answer]
It seems like this is the devtools console's fault. If you close it, and then drop files the coordinate are correct (reopen it afterwards to check).

Feel free to open an actual github issue for this but i'm not sure if we can really fix that.

**Reply by shi-yan** [reply]
yes, thank you. I confirm, it's the debugger

**Comment by yejimeiming** [comment]
+1
